sopa/embedding/models/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .resnet import Resnet50Features
+from .histoSSL import histoSSLFeatures
```

Comment:
For consistency, I would rename the module `histo_ssl`, and the class `HistoSSLFeatures`, what do you think?